CONTRIBUTING.md

```diff
@@ -80,7 +80,8 @@ On web, this means do not add any third party libraries such as Firebase or Loda
 ### Branches
 
 * [`main`](https://github.com/ionic-team/capacitor/tree/main): Latest Capacitor development branch
-* [`2.x`](https://github.com/ionic-team/capacitor/tree/2.x): Capacitor 2 (bug and security fixes only)
+* [`3.x`](https://github.com/ionic-team/capacitor/tree/2.x): Capacitor 3 (bug and security fixes only)
```

Comment:
we are not doing bug fixes, so not sure if it should say "security fixes only", or if we should be backporting bug fixes